pyproject.toml

```diff
@@ -33,7 +33,7 @@ classifiers = [  #! Update me
 "epigraphhub-config" = "epigraphhub.utils._config:create_file"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.11"
+python = "3.9.1"
```

Comment:
WARNING! This is still draft PR, not sure if packages will break because of dependency problems. Python 3.10 breaks the library.

Reply:
btw, for now don't worry to much with the docker image, we need to add tests on CI (I will work on that)